src/libraries/Common/src/System/HexConverter.cs

```diff
@@ -288,8 +288,33 @@ public static int FromLowerChar(int c)
         }
 
         [MethodImpl(MethodImplOptions.AggressiveInlining)]
-        public static bool IsHexChar(int c)
+        public static unsafe bool IsHexChar(int c)
         {
+            if (sizeof(IntPtr) == 8)
```

Comment:
Prefer `IntPtr.Size` rather than `sizeof(IntPtr)` here. It allows illink substitution to kick in. See, e.g., https://github.com/dotnet/runtime/blob/74cafe71de68fac46cd4956da98387bce9e1043c/src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.64bit.xml.

Reply:
Oh, TIL. Done in 78bc559532402d2d4dd1a2e9a85b9e9efd4240a5 🙂